UMS.launch

```diff
@@ -11,7 +11,7 @@
     </listAttribute>
     <stringAttribute key="org.eclipse.jdt.launching.CLASSPATH_PROVIDER" value="org.eclipse.m2e.launchconfig.classpathProvider"/>
     <stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="net.pms.PMS"/>
-    <stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="ums"/>
+    <stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="UniversalMediaServer"/>
```

Comment:
This line seems to go back and forth, like here https://github.com/UniversalMediaServer/UniversalMediaServer/commit/4adae84f2300e24d3a6820543e9a169bc9cef21c
@ik666 @threedguru can you two figure it out? I don't use Eclipse

Reply:
That attribute needs to have a value that is the same as the name of the eclipse project (which is the same as the folder name of the project). My eclipse project is called UniversalMediaServer, I guess someone else has named theirs ums.  The attribute was set to the value UniversalMediaServer when I first started working on UMS earlier this year - someone must have changed it to match their project - and I find that I need to keep changing it back or I cannot run UMS in Eclipse as it complains about the project being missing. Obviously, the eclipse project name needs to be standardised - to UniversalMediaServer (I would suggest!).

Reply:
I see, that makes sense to me. Can you rename yours to UniversalMediaServer @ik666 ?